demystifying_large_language_models/demystifying_large_language_models.md

```diff
@@ -0,0 +1,630 @@
+<!--
+
+author:   Joy Payton
+email:    paytonk@chop.edu
+version:  1.0.0
+current_version_description: Initial version
+module_type: standard
+docs_version: 1.2.1
+language: en
+narrator: US English Female
+mode: Textbook
+
+title: Demystifying Large Language Models
+
+comment:  Learn about large language models (LLM) like ChatGPT.
+
+long_description: There's lots of talk these days about large language models in academia, research, and medical circles.  What is a large language model, what can it actually do, and how might LLMs impact your career?  Learn more here!
+
+estimated_time_in_minutes: 60
+
+@pre_reqs
+None.  
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- Define "large language model" (LLM) 
+- Give a brief description of n-grams and word vectors
+- Give a brief description of a neural network
+- Give one example of a task that an LLM could do that could advance a biomedical project or career
+- Give one example of a caveat or pitfall to be aware of when using an LLM
+
+@end
+
+@version_history 
+ 
+@end
+
```

Comment:
```suggestion

good_first_module: false 
```

Feel free to change false to true if you want -- I don't really have strong opinions about how we're determining what a good first module is. From docs: "If this was a learner's very first experience with DART, would you be happy with this module being their first impression?" 

Required for all modules."